js/exporters/exporter-csv.js

```diff
@@ -34,33 +34,50 @@ PassmanExporter.csv.export = function (credentials, FileService, EncryptService,
 	/** global: C_Promise */
 	return new C_Promise(function () {
 		PassmanExporter.getCredentialsWithFiles(credentials, FileService, EncryptService, _log, $translate).then((function(){
-		    var headers = ['label', 'username', 'password', 'email', 'description', 'tags', 'url', 'custom_fields', 'files'];
-		    var file_data = '"' + headers.join('","') + '"\n';
-		    for (var i = 0; i < credentials.length; i++) {
-			    var _credential = credentials[i];
-			    var row_data = [];
-			    for (var h = 0; h < headers.length; h++) {
-				    var field = headers[h];
+		    const headers = [
+				'label',
+				'description',
+				'created',
+				'changed',
+				'tags',
+				'email',
+				'icon',
+				'username',
+				'password',
+				'url',
+				'renew_interval',
+				'expire_time',
+				'delete_time',
+				'files',
+				'custom_fields',
+				'otp',
+				'compromised',
+				'hidden'
+			];
+		    let file_data = '"' + headers.join('","') + '"\n';
+		    for (let i = 0; i < credentials.length; i++) {
+			    const _credential = credentials[i];
+			    let row_data = [];
+			    for (const field of headers) {
 				    if (field === 'tags') {
-					    var _tags = [];
-					    for (var t = 0; t < _credential[field].length; t++) {
-						    _tags.push(_credential[field][t].text);
+					    let _tags = [];
+					    for (const tag_field of _credential[field]) {
+							_tags.push(tag_field.text);
 					    }
```

Comment:
json.stringify may look cleaner here

Reply:
sure, but I want to preserve compatibility to the previous csv export logic. otherwise we would have another option of tag format

Reply:
Makes sense then :)